### PR TITLE
Feature/241 profile rewrite

### PR DESCRIPTION
**Pull request by flaussinot**

### CODE REVIEW

File: src/main/java/org/oasis_eu/portal/core/model/catalog/CatalogEntry.java [hunk 1]
```diff
@@ -86,6 +86,9 @@
     @JsonProperty("visible")
     private boolean visible = false;
 
+    @JsonProperty("visibility")
```
Comment (bobeal) [inline src/main/java/org/oasis_eu/portal/core/model/catalog/CatalogEntry.java]: Ouch, visibility and access_control modifications should be in another feature branch (this is not related to profile rewrite)

File: src/main/resources/public/jsx/appmanagement/service-settings.jsx.js [hunk 2]
```diff
@@ -86,12 +86,7 @@ var Service = React.createClass({
             links.push(
                 <span key={this.props.service.service.id + '-settings'} className="pull-right action-icon" onClick={this.settings}>
                     <i className="fa fa-cog fa-lg"></i>
-                </span>
-            );
-        }
-
-        if (!this.state.saved_service.service.visible && this.props.status !== 'STOPPED') {
-            links.push(
+                </span>,
```
Comment (bobeal) [inline src/main/resources/public/jsx/appmanagement/service-settings.jsx.js]: And also this one.

Wasn't the condition on STOPPED necessary ?

Comment (flaussinot) [inline src/main/resources/public/jsx/appmanagement/service-settings.jsx.js]: Yes it is necessary.
But there is a refactoring code and the 2 "<span>" are in the same "link.push(" with the same condition and only separate by a comma.
